packages/app/src/app/pages/Search/index.tsx

```diff
@@ -3,46 +3,50 @@ import {
   ALGOLIA_APPLICATION_ID,
   ALGOLIA_DEFAULT_INDEX,
 } from '@codesandbox/common/lib/utils/config';
-import { Helmet } from 'react-helmet';
 import MaxWidth from '@codesandbox/common/lib/components/flex/MaxWidth';
 import Margin from '@codesandbox/common/lib/components/spacing/Margin';
+import { History, Location } from 'history';
 import qs from 'qs';
-import React, { useCallback, useEffect, useRef, useState } from 'react';
+import React, {
+  FunctionComponent,
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import { Helmet } from 'react-helmet';
 import {
+  Configure,
   InstantSearch,
-  SearchBox,
   PoweredBy,
-  Configure,
+  SearchBox,
 } from 'react-instantsearch/dom';
-import { History, Location } from 'history';
 
-import { Navigation } from 'app/pages/common/Navigation';
 import { useOvermind } from 'app/overmind';
+import { Navigation } from 'app/pages/common/Navigation';
 
 import 'instantsearch.css/themes/reset.css';
 
-import { Container, Content, Main, StyledTitle } from './elements';
+import { Container, Content, Main, Title } from './elements';
 import Filters from './Filters';
 import Results from './Results';
 import Styles from './search';
 
-interface ISearchProps {
```

Comment:
We Prefer interface over type for typing Props as mentioned in "Best Practices and Style Guide" section. So `ISearchProps` interface was fine

Reply:
Yeah, why was this deleted?